test/helper/TestFactory.js

```diff
@@ -118,11 +118,12 @@ export class TestFactory {
     this.cryptography_repository = new CryptographyRepository(this.cryptography_service, this.storage_repository);
     this.cryptography_repository.currentClient = ko.observable(currentClient);
 
-    if (mockCryptobox) {
+    if (mockCryptobox === true) {
```

Comment:
was that an issue?

Reply:
I want to be sure to check against a boolean. If `mockCryptobox` would be `'a'` the condition would still be valid but it should not.